src/box/lua/net_box.c

```diff
@@ -820,6 +830,26 @@ netbox_encode_select(lua_State *L, int idx, struct mpstream *stream,
 	mpstream_encode_uint(stream, IPROTO_LIMIT);
 	mpstream_encode_uint(stream, limit);
 
+	/* encode after */
+	if (lua_isstring(L, idx + 6)) {
+		mpstream_encode_uint(stream, IPROTO_AFTER_POSITION);
+		size_t size;
+		const char *pos = lua_tolstring(L, idx + 6, &size);
+		char *buf = mpstream_reserve(stream, size);
```

Comment:
There's `mpstream_encode_binl` and `mpstream_memcpy`.

Reply:
Using `mpstream_memcpy`.